### ISSUE TITLE
phenio drops xrefs

### ISSUE THREAD

**Issue by cmungall** [monarch-initiative/phenio#91]
```
✗ sqlite3 $db/hp.db
SQLite version 3.37.0 2021-12-09 01:34:53
Enter ".help" for usage hints.
sqlite> select * from statements where subject='HP:0030376' and predicate='oio:hasDbXref';
HP:0030376|HP:0030376|oio:hasDbXref||UMLS:C4072915|||
sqlite> ^D
```

```
✗ sqlite3 $db/phenio.db
SQLite version 3.37.0 2021-12-09 01:34:53
Enter ".help" for usage hints.
sqlite> select * from statements where subject='HP:0030376' and predicate='oio:hasDbXref';

```

**Comment by cmungall** [comment]
need it to show up here https://monarchinitiative.org/HP:0030376

**Comment by matentzn** [comment]
This is due to the fact the HPO enters PHENIO via uPheno, which is not very modular; I am changing this and adding all phenotype ontologies as individual components to phenio now

**Comment by matentzn** [comment]
This will be addressed by #94
